scripts/src/java/org/oppia/android/scripts/common/RepositoryFile.kt

```diff
@@ -42,33 +42,25 @@ class RepositoryFile() {
     fun collectSearchFiles(
       repoPath: String,
       expectedExtension: String = "",
-      exemptionsList: List<String> = listOf<String>()
+      exemptionsList: List<String> = listOf()
     ): List<File> {
       // Note that Files.walk() is used instead of Kotlin's walk() function since the latter follows
       // symbolic links which is almost 10x slower than not following them (due to very deep Bazel
       // build directories), and it's not necessary to follow the symlinks.
-      return Files.walk(File(repoPath).toPath()).asSequence().map(Path::toFile).filter { file ->
-        val isProhibited = checkIfProhibitedFile(retrieveRelativeFilePath(file, repoPath))
-        !isProhibited &&
+      val repoRoot = File(repoPath).absoluteFile.normalize()
+      return Files.walk(repoRoot.toPath()).asSequence().map(Path::toFile).map { file ->
+        file.absoluteFile.normalize()
+      }.filter { file ->
```

Comment:
TODO: mention that there are some basic cleanup efforts here to simplify the exemption checking a bit. Shouldn't change callsite behavior.

Reply:
Done.